blockchain/block.go

```diff
@@ -9,10 +9,9 @@ import (
 )
 
 const (
-	// BlockSize is the maximum size of a block in bytes when marshaled (about 250K).
-	BlockSize = 1 << 18
-	// BlockHeaderLen is the length in bytes of a block header.
-	BlockHeaderLen = 2*(32/8) + 64/8 + 2*HashLen
+	// MaxBlockSize is the maximum size of a block in bytes when marshaled
+	// (about 250K).
+	MaxBlockSize = 1 << 18
```

Comment:
Block size is busy blowing the bitcoin universe up - are we sure we want to enforce it? https://medium.com/@olivierjanss/the-solution-to-the-block-size-debate-42c3814ff481
https://www.bitcoinunlimited.info/

Did you guys discuss this on Saturday? I'm game if we're consciously making the decision.